MD/md-38/README.md

```diff
@@ -0,0 +1,76 @@
+
+  2. **Accountability**: By noting errors openly, we maintain a high level of responsibility and ownership over our content. It’s an affirmation that we value precision and are ready to correct oversights.
+
+  Each erratum should briefly describe the discrepancy and the correction made, accompanied by a reference to the date and version of the desiderata in which the error was identified.
+
+  TODO: Maintain this comment.
+-->
+
+## Appendix
+
+### A1: Suggested Solutions for Fixed Token Supply
+
+| Name              | Description                           | Gas Destination       | Reward Form          | Challenges                          |
+|-------------------|---------------------------------------|-----------------------|----------------------|-------------------------------------|
+| **Decoupled**   | • Gas spent on the L2 deposited to pools on the L2. <br> • Rewards on the L1 are issued in a LP token which is demurrage. <br> • Ideally, intermediary tokens are used in the bridge to shield gas and staking token from bridge fallibility. | Pool on the L2         | FFS LP Token; voucher periods can be assigned. | • LP Token's demurrage assigns a lifetime to the viability of the settlement protocol which will require a new or asymptotic token voucher or token generation event at some point. |
+| **Trickle-back** | • Gas spent on the L2 recirculates to pools on the L2. <br> • Rewards are provided on the L2 for bridging back to the L1 at the address of the FFS LP Token. <br> • Ideally, intermediary tokens are used in the bridge to shield gas and staking token from bridge fallibility. | Pool on the L2         | Staking Token               | • Rewards for pool providers to bridge back to the L1 at the address of the FFS LP Token need to be sufficient. <br> • Pool providers likely need to have only custodial access to token, otherwise intended rewarding game theory would likely be intractable. |
+| **Settle with Gas**            | • The settlement messages include the amount of gas spent. The LP for the FFS claims token locked in the L1 bridge. <br> • Bridge may still benefit from intermediary tokens.                              | FFS LP Token        | Staking Token               | • Unclear how to handle forks. <br> • Centralization remains as FFS contract needs to have capability to claim token locked in L1 bridge. However, this can be governed on-chain in this case. <br> • Intermediary tokens can still help reduce attack surface via the bridge. Now attacks would likely focus on exploiting capabilities of the settlement contract to unlock token. |
+| **Noble Gasmaster**      | • A trusted signer is responsible for successfully relaying gas spent. The use of intermediary tokens in the bridge is moot.              | FFS LP Token         | Staking Token             | • Centralized signer.   |
```

Comment:
similar to the "acceptor" is there a way how any validator can be permitted to transfer L2 pool tokens to the L1 reward contract? (at the cost of L2fees and L1fees for that validator)? 

Similar to the Acceptor model, a L1-selected acceptor could be responsible for transferring rewards from L2 to L1, and get rewarded on L1 for this, however a volunteer could transfer also the L2RewardPool balance to the L1RewardPool balance (at their own cost)